front/app/components/LandingPages/citizen/EventsWidget.tsx

```diff
@@ -135,17 +115,7 @@ const EventsWidget = ({ staticPageId }: Props) => {
               </VerticalCenterer>
             )}
 
-            {!isNilOrError(events) && events.data.length > 0 && (
-              <CardsContainer>
-                {events.data.map((event) => (
-                  <StyledEventCard
-                    event={event}
-                    key={event.id}
-                    titleFontSize={18}
```

Comment:
The default font-size already seems to be 18px. Besides, the "px" part was not specified so this wasn't doing anything.